packs/feats/briny-beverage.json

```diff
@@ -28,7 +28,22 @@
             "remaster": true,
             "title": "Pathfinder Lost Omens: Tian Xia Character Guide"
         },
-        "rules": [],
+        "rules": [
```

Comment:
Would this be better as a self effect?

Reply:
It could be. The last sentence there about the specific body of water it works from gave me pause.